### ISSUE TITLE
MSDF fonts perform pre-multiplied alpha, standard fonts don't

### ISSUE THREAD

**Issue by ZigZaggh** [JujuAdams/Scribble#199]
Hi I was working with the MSDF fonts and ran in to ".blend(colour, alpha)" not working as expected, I tested with a "normal" font, and that works.

It looks like it just blends with Black not matter what color you are setting.

I'm testing on Windows, with the latest stable GMS IDE and Runtime version, Scribble version 7.1.2pre002.

Sincerely

**Comment by JujuAdams** [comment]
Could you show me what code you're using please?

**Comment by ZigZaggh** [comment]
Just for testing I'm doing the following:

``` GML
scribble("[Font12][c_red]88888").blend(c_white,` 1).draw(500, 240);
scribble("[Font12][c_red]88888").blend(c_white, 0.75).draw(500, 280);
scribble("[Font12][c_red]88888").blend(c_white, 0.5).draw(500, 320);
scribble("[Font12][c_red]88888").blend(c_white, 0.25).draw(500, 360);

scribble("[MSDF_Roboto_Medium][c_red]88888").blend(c_white, 1).draw(600, 240);
scribble("[MSDF_Roboto_Medium][c_red]88888").blend(c_white, 0.75).draw(600, 280);
scribble("[MSDF_Roboto_Medium][c_red]88888").blend(c_white, 0.5).draw(600, 320);
scribble("[MSDF_Roboto_Medium][c_red]88888").blend(c_white, 0.25).draw(600, 360);
```


And the attached picture is showing the result, left column "normal" font right is MSDF font.
![Screenshot 2021-03-27 114855](https://user-images.githubusercontent.com/43120666/112718341-ad8f1f80-8ef2-11eb-9fe0-a9bcc53764fc.png)

**Comment by JujuAdams** [comment]
Huh, for some reason MSDF fonts are doing pre-multiplied alpha. Not sure why I did that! Easy fix though - comment out line 54 in the fragment shader for `__shd_scribble_msdf`. I'll put out a release with that in at some point

**Comment by ZigZaggh** [comment]
If I comment out the line 54 I just get no blending at all!?
I'm not much into shaders, so I'm not completely sure what is happening :)

**Comment by JujuAdams** [comment]
*narrows keys*

Yeah ok lol that makes sense. Try `gl_FragColor.a *= v_vColour.a;` instead.

**Comment by ZigZaggh** [comment]
Works perfectly, thank you so much for the fast reply Juju 👌 😃

**Comment by JujuAdams** [comment]
Implemented publicly for v7.1.2
